errors.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"fmt"
+)
+
+// chanCreationError is an enum which describes the exact nature of an error
+// encountered when a user attempts to create a channel with the faucet. This
+// enum is used within the templates to determine at which input item the error
+// occurred  and also to generate an error string unique to the error.
+type ChanCreationError uint8
+
+const (
+	// 0 NoError is the default error which indicates either the form hasn't
+	// yet been submitted or no errors have arisen.
```

Comment:
These are not following the go doc convention. The first word should be the name of the of the constant. There's no need to document the value as it should show in the respective pkg.go.dev page.